### ISSUE TITLE
System requirements for PHP are confusing (8.0 vs. 8.1)

### ISSUE THREAD

**Issue by froschdesign** [sulu/skeleton#217]
| Q | A
| --- | ---
| Bug? | maybe
| New Feature? | no
| Sulu Version | 2.5.x / latest
| Browser Version | irrelevant

#### Actual Behavior

The skeleton requires PHP version 8.1 and higher:

https://github.com/sulu/skeleton/blob/eb5158cc1cff3a6c3634364d03e143b44e99283b/composer.json#L29-L30

Sulu itself allows version 8.0 and higher:

https://github.com/sulu/sulu/blob/097a23c376f1fddbcdfdbd4f564f7847ec0cc934/composer.json#L27-L28

The documentation says PHP 8.0 and higher:

> The following requirements must be met to run Sulu:
> 
> - Mac OSX, Linux or Windows
> - [Apache](https://docs.sulu.io/en/2.5/cookbook/web-server/apache.html) or [Nginx](https://docs.sulu.io/en/2.5/cookbook/web-server/nginx.html) with enabled URL rewriting
> - **[PHP](http://php.net/) 8.0 or higher**
> - …

https://docs.sulu.io/en/2.5/reference/requirements.html

#### Expected Behavior

If Sulu allows version 8 of PHP and the documentation also lists this version under the system requirements, then the skeleton should allow the same version.

#### Steps to Reproduce

See above.

#### Possible Solutions

Update the Composer configuration of this package.

**Comment by alexander-schranz** [comment]
Thank you for the feedback.

Why `sulu/sulu` still supports `PHP 8.0`. The `skeleton` and new installation is shipped with latest Symfony Version which requires atleast `PHP 8.1`. So at current state you can use PHP 8.0 to run latest version of Sulu and old projects on PHP 8 can update to that latest version, but you can not start a new project based on the skeleton because Symfony 6.1 (6.2 is used in 2.5 skeleton) requires now PHP 8.1: https://symfony.com/blog/symfony-6-1-will-require-php-8-1

So looking here I think we should update the docs to set the requirement to PHP 8.1 there for Sulu 2.5. Do you want to create a pull request for it?

**Comment by froschdesign** [comment]
I found another notice:

> ## ✅&nbsp; Requirements
> 
> Sulu requires a **PHP version higher or equal to 8.0** and is compatible with every **Symfony version starting from 5.4**. Have a look at the `require` section in the [composer.json](composer.json) of the [sulu/sulu](https://github.com/sulu/sulu) core framework to find an **up-to-date list of the requirements** of Sulu content management system.

https://github.com/sulu/sulu#-requirements

**Comment by froschdesign** [comment]
> Why `sulu/sulu` still supports `PHP 8.0`. The `skeleton` and new installation is shipped with latest Symfony Version which requires atleast `PHP 8.1`.

Okay, that explains it, but it's not clear when you read the documentation, because it says version 8.0 and refers to the skeleton.

> Do you want to create a pull request for it?

I will create a pull request on https://github.com/sulu/sulu-docs

**Comment by alexander-schranz** [comment]
Yeah think change the docs that requirement is PHP 8.1 make sense as people reading that are mostly the ones which starting a new project and not working on an existing one.

**Comment by alexander-schranz** [comment]
fixed as sulu/sulu and sulu/skeleton are now both on 8.2 why we still recommend create new projects with 8.4 or 8.5. Thx for the feedback.
